cachito/workers/tasks/general.py

```diff
@@ -169,13 +170,25 @@ def save_bundle_archive_checksum(request_id: int) -> None:
     bundle_dir.bundle_archive_checksum.write_text(checksum, encoding="utf-8")
 
 
-@app.task(priority=10)
+@app.task(priority=5)
```

Comment:
5 is the default: https://github.com/release-engineering/cachito/blob/9a42d2b5b31b9644cd1d5963a4ec4e359cd9d2af/cachito/workers/config.py#L84-L85

It looks like no other task defines priority=5 explicitly, it may be good to keep it that way for consistency

Reply:
:man_facepalming: